packages/terra-clinical-item-collection/tests/wdio/clinical-item-collection-spec.js

```diff
@@ -27,10 +29,14 @@ Terra.describeViewports('Item Collection', ['tiny', 'small', 'medium', 'large',
     before(() => {
       browser.url('/#/raw/tests/terra-clinical-item-collection/clinical-item-collection/item-collection-selectable');
       browser.waitForValue('#item1');
-      browser.click('#item1');
+      browser.leftClick('#item1'); // clicking a table row element in firefox is a bug. browser.leftClick() uses W3C actions.
       browser.click('#selected-key');
     });
 
+    // TODO: remove disabled color-contrast rule: https://github.com/cerner/terra-clinical/issues/673
+    const ignoreColorContrastRule = { 'color-contrast': { enabled: browserName !== 'firefox' } };
+    Terra.it.validatesElement({ rules: ignoreColorContrastRule });
+
     Terra.it.validatesElement();
```

Comment:
Does this validatesElement need to be removed ?

Reply:
Good call. Not sure how I missed that: https://github.com/cerner/terra-clinical/pull/674/commits/883af7fbe265817a02063210a1fb7687af7a78b4